Resources/Prototypes/Entities/buildings/turret.yml

```diff
@@ -1,6 +1,7 @@
 - type: entity
   id: TurretBase
   name: Turret Base
+  abstract: true
```

Comment:
Uhhh, I think this is the BASE of the turret, not the base type of a turret

Reply:
oh, I made it abstract because it showed without an icon in the spawnlist.. But if that's the case I'll remove that line

Reply:
Yeah, I checked and indeed you were right, so I removed the abstract line!

Reply:
Yeah turrets aren't supposed to be spawned split like that, since they're a 2-entity... entity (A 'Base' and a 'Top', we have 1 base and 2 tops (Spotlight, Gun))
Does abstract do anything other than hiding from the spawn list? if not then it probably WAS correct to hide it.